cdap-ui/app/wrangler/components/Wrangler/index.js

```diff
@@ -219,6 +306,49 @@ export default class Wrangler extends Component {
     );
   }
 
+  applyToHydrator() {
```

Comment:
Not sure if this is `applyToHydrator` all the time. Its returning the properties and schema right. Probably `getWranglerOutput`?